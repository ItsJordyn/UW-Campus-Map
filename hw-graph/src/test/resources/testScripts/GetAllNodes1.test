# GetAllNodes1 testing file
# Tests that all the nodes in a graph can be printed out
# In this case (1 node).

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# No edge created

# Print the nodes in the graph
ListNodes graph1