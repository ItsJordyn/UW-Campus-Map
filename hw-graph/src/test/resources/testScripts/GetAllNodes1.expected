# GetAllNodes1 testing file
# Tests that all the nodes in a graph can be printed out
# In this case (1 node).

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# No edge created

# Print the nodes in the graph
graph1 contains: n1